docs/source/quantization.rst

```diff
@@ -129,7 +128,7 @@ Note that for FX quantization, the corresponding functionals are also supported.
 |nn.EmbeddingBag            | Y (activations    |                    |
 |                           | are in fp32)      | Y                  |
 +---------------------------+-------------------+--------------------+
-|nn.Embedding               | Y                 | N                  |
+|nn.Embedding               | Y                 | Y                  |
```

Comment:
could you add ("float qparams weight only") in the Dynamic column:

```
Y (float qparams weight only)
```

Reply:
isn't it fp32 activation for both embedding and embedding bag for both static and dynamic? It seems like it should be Y (float qparams weight only) for all 4 entries for both fx and eager mode.